lib/plugin.c

```diff
@@ -314,6 +317,19 @@ plugin_dlopen_module_on_path(const gchar *module_name, const gchar *module_path)
       if (is_file_regular(plugin_module_name))
         break;
 #endif
+#ifdef __APPLE__
+      /* On Mac the modules are in .dylib files */
+      dot = strrchr(plugin_module_name, '.');
+      if (dot)
+        {
+          *dot = 0;
+          p = g_strdup_printf("%s.dylib", plugin_module_name);
```

Comment:
I think those 3 cases could be replaced with this one:
```C
       dot = strrchr(plugin_module_name, '.');
       if (dot)
         {
           *dot = 0;
           p = g_strdup_printf("%s.%s", plugin_module_name, G_MODULE_SUFFIX);
           g_free(plugin_module_name);
           plugin_module_name = p;
         }
       if (is_file_regular(plugin_module_name))
         break;
```

Reply:
Yeah, that would do it too.  Thought about that just after submitting this...

Reply:
Could you update the PR with that version ?
The current version executes both `la` and `dylib` detection in case of apple.